src/components/forms/pattern-form/pattern-form.js

```diff
@@ -181,76 +193,90 @@ const PatternForm = ({ pattern, id, isEdit }) => {
-                  <div className={styles.inputError}>
-                    {errors.patternConstructorImage}
-                  </div>
-                )}
-              </div>
-            </div>
+          <Grid item xs={12}>
+            <Paper className={styles.patternItemUpdate}>
+              <div>
+                <span className={styles.imageUpload}>
+                  {config.labels.pattern.avatarText}
+                </span>
+                <div className={styles.imageUploadAvatar}>
+                  <ImageUploadContainer handler={handleLoadMainImage} />
+                  {patternImage && (
+                    <Avatar src={patternImage}>
+                      <Image />
+                    </Avatar>
+                  )}
+                  {touched.patternImage && errors.patternImage && (
+                    <div className={styles.inputError}>
+                      {errors.patternImage}
+                    </div>
+                  )}
+                </div>
 
-            <TextField
-              data-cy='material'
-              id='material'
-              className={styles.textField}
-              variant='outlined'
-              label={config.labels.pattern.material}
-              value={values.material}
-              onChange={handleChange}
-              error={touched.material && !!errors.material}
-            />
-            {touched.material && errors.material && (
-              <div data-cy='material-error' className={styles.inputError}>
-                {errors.material}
+                <span className={styles.imageUpload}>
+                  {config.labels.pattern.constructorImgText}
+                </span>
+                <div className={styles.imageUploadAvatar}>
+                  <ImageUploadContainer handler={handleLoadConstructorImage} />
+                  {constructorImg && (
+                    <Avatar src={constructorImg}>
+                      <Image />
+                    </Avatar>
+                  )}
+                  {touched.patternConstructorImage &&
+                    errors.patternConstructorImage && (
+                    <div className={styles.inputError}>
+                      {errors.patternConstructorImage}
+                    </div>
+                  )}
+                </div>
               </div>
-            )}
-          </Paper>
-        </Grid>
-        {map(languages, (lang) => (
-          <LanguagePanel lang={lang} inputOptions={inputOptions} key={lang} />
-        ))}
-        <BackButton />
-        <SaveButton
-          className={styles.saveButton}
-          data-cy='save-btn'
-          type='submit'
-          title={SAVE_TITLE}
-          values={values}
-          errors={errors}
-        />
-      </form>
+              <FormControl
+                variant='outlined'
+                className={`${styles.formControl} ${styles.materialSelect}`}
+              >
+                <InputLabel variant='filled'>Матеріал</InputLabel>
```

Comment:
Please move this magic string to separate file in `config `folder.

Reply:
done